flow/cli/project.go

```diff
@@ -153,6 +140,14 @@ func (p *Project) ContractConflictExists(network string) bool {
 	return len(all) != len(uniq)
 }
 
+func (p *Project) HostWithOverride(host string) string {
```

Comment:
Maybe makes sense to later update this to `HostWithOverride(network, hostOverride string)`